test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -167,10 +124,14 @@ void fillRandomTest(std::function<void(json_spirit::mValue&, bool)> _doTests, st
 		dev::test::RandomCode::parseTestWithTypes(newTest, nullReplaceMap);
 		json_spirit::read_string(newTest, v);
 		_doTests(v, true);
+		for (auto& obj: v.get_obj())
+			if (!obj.second.get_obj().count("post"))
+				BOOST_ERROR("An error occured when executing random state test!");
+		dev::test::addClientInfo(v, "random generated test");
 	}
 	catch(...)
 	{
-		std::cerr << "Test fill exception!";
+		BOOST_ERROR("Error generating random test!");
```

Comment:
Which exceptions can happen here? You could display some info from exception object instead of just "unkown error" message